Test.Microsoft.NET.Build.Containers.Filesystem/TargetsTests.cs

```diff
@@ -0,0 +1,87 @@
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Locator;
+
+namespace Test.Microsoft.NET.Build.Containers.Targets;
+
+[TestClass]
+public class TargetsTests
+{
+    private static string CombinedTargetsLocation;
+
+    private static string CombineFiles(string propsFile, string targetsFile)
+    {
+        var propsContent = File.ReadAllLines(propsFile);
+        var targetsContent = File.ReadAllLines(targetsFile);
+        var combinedContent = new List<string>();
+        combinedContent.AddRange(propsContent[..^1]);
```

Comment:
TIL! https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/member-access-operators#index-from-end-operator-

Reply:
yeah man, slicing is the bees knees :)